packages/components/src/utils/browser.ts

```diff
@@ -0,0 +1,10 @@
+type Browser = 'Safari' | 'others'
+
+const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent)
+
+export const isBrowser = (browser: Browser): boolean => {
```

Comment:
maybe better name would be isBrowser-> isSafariBrowser

Reply:
it is only internal for that file